### PR TITLE
Add _cat functions to readme

### PR DESCRIPTION
**Pull request by maude-v**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -52,75 +52,76 @@ All the functions in the elicitr package start with the prefix `elic_`. After th
 #### How elicitr works
 
 Just like you create a form to collect estimates in an elicitation process, the core of elicitr is the creation of an object to store the metadata information. This allows to check whether experts have given their answers in the expected way.\
-Any analysis starts by creating this object with the `start` function. Then, data can be added and retrieved using the `add_data` and `get_data` functions respectively. Finally data can be plotted using the `plot` function. Details about the implementation and example usages of these functions can be seen bellow.
+Any analysis starts by creating this object with the `start` function. Then, data can be added and retrieved using the `add_data` and `get_data` functions respectively. Finally data can be plotted using the `plot` function. Details about the implementation and example usages of these functions based on the simulated datasets inluded in elicitr can be seen bellow.\
+Their are five simulated datasets provided in elictr. They are intended to demonstrate the functionality of the package and do not represent an actual elicitation process (names are also randomly generated).
```
Comment (sgvignali) [inline README.Rmd:56]: Line 56: change "Their are five simulated datasets provided in elictr." with "In total, there are five simulated datasets provided with the package."

File: README.Rmd [hunk 2]
```diff
@@ -133,7 +134,89 @@ The message printed when the function is ran informs users that the minimum and
...
+                                     experts = 6,
+                                     mechanisms = c("mechanism_1","mechanism_2", "mechanism_3"),
+                                     title = "Categorical elicitation example")
+```
+
+```{r}
+my_elicitation_cat
+```
+
+#### Functions under development
```
Comment (sgvignali) [inline README.Rmd:174]: I would remove all the text in this section. First, there are still many references to the elic_cont variables. Second, we did not write these function yet, so we cannot provide examples.

I would simply list the function under development, e.g.:

* `elic_cat_get_data()`
* `elic_cat_plot()`

**Review by sgvignali — CHANGES_REQUESTED** [review]
I have only 2 comments, could you please address them?

File: README.Rmd (hunk 1 above)
Comment (maude-v) [reply README.Rmd:56]: thank you for seing the typo (their/there). I think the correct form is provided "in" the package and not "with" the package.

Comment (sgvignali) [reply README.Rmd]: Either "provided with" or "included in", but "provided in".

File: README.Rmd [hunk 3]
```diff
@@ -52,75 +52,76 @@ All the functions in the elicitr package start with the prefix `elic_`. After th
...
 #### Simulated datasets
 
-Two simulated datasets are included in elicitr. These datasets are intended to demonstrate the functionality of the package and do not represent an actual elicitation process (names are also randomly generated).
+Two datasets that simulate the elicitation of continuous variables are included in elicitr:
 
 ```{r}
 round_1
 ```
-###
+####
```
Comment (sgvignali) [inline README.Rmd:67]: Please, remove the pound sign.

File: README.Rmd [hunk 4]
```diff
@@ -133,7 +134,50 @@ The message printed when the function is ran informs users that the minimum and
...
 
-#### In development
+#### Simulated datasets
+
+Three datasets that simulate the elicitation of categorical variables are included in elicitr:
+
+```{r}
+mechanism_1
+```
+###
```
Comment (sgvignali) [inline README.Rmd:144]: Please, remove the pound signs.

File: README.Rmd [hunk 5]
```diff
@@ -133,7 +134,50 @@ The message printed when the function is ran informs users that the minimum and
...
+```{r}
+mechanism_2
+```
+###
```
Comment (sgvignali) [inline README.Rmd:148]: Please, remove the pound signs.

**Review by sgvignali — CHANGES_REQUESTED** [review]

**Pull request merged**